src/assets/data/schema/schemas.json

```diff
@@ -0,0 +1,55 @@
+{
+	"schemas": [
+		{
+		"active_questionnaire_phq8_Schema": {
+			"namespace": "org.radarcns.questionnaire",
+			"type": "record",
+			"name": "Phq",
+			"doc": "Phq questionnaire",
+			"fields": [{
+					"name": "questionnaireId",
+					"type": "int",
+					"doc": "Questionnaire identifier"
+				},
+				{
+					"name": "answers",
+					"type": {
+						"type": "map",
```

Comment:
Instead of a `map`, it would be better to use an `array`. The approach could be storing in the entry `0` the answer of question `1`, in entry `1` answer from question `2` and so on.